duckduckgo/iOS
# Remove double search header

Task/Issue URL: https://app.asana.com/0/856498667320406/1194658054793361
Tech Design URL: https://app.asana.com/0/856498667320406/1199403269515656
CC:

**Description**:
Experiment to remove search header from the SERP and use in-app URL bar for re-querying.

**Steps to test this PR**:

Ensure non-experiment variant works without any change.

For experiment:
- URL bar should highlight all text only on non-SERP tabs or after user comes back to the app. Otherwise text is not highlighted and autocomplete is presented.
- Support for RQ pixels (see tech design).
- Videos, Images and News verticals are persisted across re-queries.
- Present Dax logo instead of grade icon on the SERP.
- SERP settings should hide header options in appearance tab.




**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
